packages/framework/data-object-base/src/runtimeFactory.ts

```diff
@@ -30,7 +30,7 @@ export class RuntimeFactory extends RuntimeFactoryHelper {
 	constructor(props: {
```

Comment:
Who uses RuntimeFactory?  Why can't we remove it?

Reply:
Not sure if anyone uses it, but I'm not so sure we can just arbitrarily remove things like this without deprecating first.